### DISCUSSION TITLE
Database broken after using API

### DISCUSSION THREAD

**Discussion by mikygee** [netbox-community/netbox#17434] [Help Wanted!]
### Deployment Type

Self-hosted

### NetBox Version

v4.0.11

### Python Version

3.10.14

### Postgresql Version

16.4

### Steps to Reproduce

Not known yet

### Expected Behavior

Netbox to work normally

### Observed Behavior

Hello I messed up using API,
I was trying to add a new platform like this
`PUT $data = '{ "name": "' . "SOMENAME .'", "slug": "' . "somename" .'", "manufacturer": 1, "description": "' . "somedesc" . '" }';`
or
`$data = '{ "name": "' . "SOMENAME" .'", "slug": "' . "somename" .'", "manufacturer": { "id": ' . "1" . ' }, "description": "' . "somedesc" . '" }';`

Now I see this message when I want to add a new platform or manufacturer
```
<class 'django.db.utils.ProgrammingError'>

relation "extras_objectchange" does not exist
LINE 1: INSERT INTO "extras_objectchange" ("time", "user_id", "user_...
```

```
# \d extras_objectchange
Did not find any relation named "extras_objectchange".
```

python3 manage.py migrate didn't help

I think it's not normal that I broke the database structure using APIs

Also how can I fix it ?

**Comment by quantum909** [comment]
See  [here](https://github.com/netbox-community/netbox/issues/17401])

**Comment by mikygee** [comment]
Hello quantum909,

Thank you for your message and pointing me this issue #17401 

Indeed, I upgraded to 4.1.0 from 4.0.11 but then moved back to 4.0.11.
However, I was coding using APIs and it worked after this upgrade/downgrade.

It seemed to me that I did and action through API that caused this.

**Comment by quantum909** [comment]
Did you tryed [this](https://github.com/netbox-community/netbox/discussions/17388#discussioncomment-10582898)

**Comment by mikygee** [comment]
Hello quantum909,
I did this command
`delete from django_migrations WHERE name='0011_move_objectchange';`
But it didn't solve the problem.

I use Postgresql 16.4 and python 3.10.14

**Comment by mikygee** [comment]
Hello,
I didn't manage to solve the problem and I made a database restauration and upgraded again to 4.0.11

**Reply by mikygee** [reply]
I did tests with my code and it seems that the problem was not my code.
